scripts/api-extractor/api-extractor.common.v-next.json

```diff
@@ -4,15 +4,31 @@
     "enabled": true
   },
   "docModel": {
-    "enabled": true,
+    "enabled": false,
     "apiJsonFilePath": "<projectFolder>/dist/<unscopedPackageName>.api.json"
   },
   "dtsRollup": {
     "enabled": true,
-    "untrimmedFilePath": "<projectFolder>/dist/index.d.ts"
+    // TODO: replace following line with these 2 commented ones after all v9 is migrated to new build and .d.ts API stripping
```

Comment:
so api extractor supports comments in JSON ? 👀

Reply:
yeah, it supports `jsonc`